components/04-molecules/article&blog-post-elements/author-bio/_author-bio.scss

```diff
@@ -35,6 +36,46 @@ $author-text-margin: 0.18rem;
     margin-bottom: $spacer-xl;
     background-color: $author-bio-background-color;
   }
+
+  //CONTACT PERSON/ BEWERBUNGSBUDDIES
+  &.author-bio--large {
+    /* .avatar--xl{
+      background-color: $brand-primary;
+      border-radius: 100%;
+    } */
+
+    .avatar__image {
+      background-color: $brand-primary;
+      resize: 50%;
+    }
+
+    .author-bio__name {
+      padding-bottom: 1%;
+      font-size: $author-name-xl;
+    }
+
+    .author-bio__text {
+      padding-top: 10%;
+      text-align: center;
+
+    > p:last-child {
+        @extend %heading-font-regular;
+
+        padding-top: 7%;
+      }
+
+    > p:first-child, .author-bio__info {
+      @extend %body-font-italic;
+
+      font-size: $font-size-base*1.45;
```

Comment:
Bitte keine Multiplikation hier verwenden. Es gibt einige vorgefertigte font-sizes, die du verwenden kannst. Diese findest du in der typoghraphy.scss